.github/workflows/maintenance-cherry-pick-workflow.yml

```diff
@@ -0,0 +1,40 @@
+name: cherry-pick on maintenance branch
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    outputs:
+      output: ${{ steps.cherry_pick.outputs.output }}
+    if: |
+      github.event.pull_request.merged == true &&
+      contains(github.event.pull_request.labels.*.name, 'target: maintenance')
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Cherry-pick changes into maintenance"
+      id: cherry_pick
+      run: |
+        git checkout maintenance
+        echo "::set-output name=output::$(git -c user.name="cherry.picker" \
```

Comment:
is it not possible to remain the original committer?

Reply:
![image](https://user-images.githubusercontent.com/3184240/160595094-347c5ba9-f0c9-4c7c-925a-1e203843473f.png)
That's how it's displayed. We can see that it was cherry-picked by the bot.